### PR TITLE
Basic functionality for MSAL automation app

### PR DESCRIPTION
**Pull request by oldalton**
Implemented basic functionality for MSAL automation app, took ADAL automation app as example and existing MSAL code

### CODE REVIEW

File: MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.m [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+//------------------------------------------------------------------------------
+
+#import "MSALAccessTokenCacheItem+Automation.h"
+#import "MSALUser+Automation.h"
+
+@implementation MSALAccessTokenCacheItem (Automation)
+
+- (NSDictionary *)msalItemAsDictionary
+{
+    NSMutableDictionary *resultDict = [NSMutableDictionary dictionary];
```
Comment (RPangrle) [inline MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.m]: Can you use the _json dictionary instead?

File: MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+#import "MSALAccessTokenCacheItem.h"
+
+@interface MSALAccessTokenCacheItem (Automation)
+
+- (NSDictionary *)msalItemAsDictionary;
```
Comment (RPangrle) [inline MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.h]: 'msal' prefix isn't needed as this is an MSAL class.

File: MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.m (hunk 1 above)
Comment (oldalton) [inline MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.m]: Done 👍 thank you for reminding about it

File: MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.h (hunk 2 above)
Comment (oldalton) [inline MSAL/test/automation/ios/util/MSALAccessTokenCacheItem+Automation.h]: done 👍

File: MSAL/test/automation/ios/util/MSALResult+Automation.m [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+#import "MSALResult+Automation.h"
+#import "MSALUser+Automation.h"
+
+@implementation MSALResult (Automation)
+
+- (NSDictionary *)msalItemAsDictionary
```
Comment (oldalton) [inline MSAL/test/automation/ios/util/MSALResult+Automation.m]: I think you had an outdated version, as this was fixed some days ago :)

**Pull request merged**